### PR TITLE
Feature(nextjs-logging-wrapper): added package

### PR DESCRIPTION
**Pull request by SamSalvatico**
### Description

Added the `nextjs-logging-wrapper` package

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**

### CODE REVIEW

**Review by Thurfjell — APPROVED** [review]

File: packages/nextjs-logging-wrapper/src/server-logger.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    logger = getPinoInstance({
+      getConfigurations: getLoggerConfiguration,
+      minLevel: minimumLevel,
+    });
+  }
+
+  return logger;
+};
+
+export const getLoggerConfiguration = (
```
Comment (edge33) [inline packages/nextjs-logging-wrapper/src/server-logger.ts:35]: `getLoggerConfiguration` looks the same on both client and server logger, if this is the case, would it make sense to externalise this config to a separate file and import it?

Comment (SamSalvatico) [reply packages/nextjs-logging-wrapper/src/server-logger.ts:35]: they are slightly different, but I can try to normalize them 🆗

Comment (edge33) [reply packages/nextjs-logging-wrapper/src/server-logger.ts]: if they are different it is not worth externalising then. thanks for sharing, LGTM

**Review by edge33 — APPROVED** [review]
LGTM

**Pull request merged**